nyc_buildings/anaconda-project.yml

```diff
@@ -8,35 +8,44 @@ examples_config:
   - philippjfr
   labels:
   - datashader
+  - hvplot
   deployments:
   - command: notebook
+  - command: dashboard
 
 user_fields: [examples_config]
 
 channels:
-- defaults
+- conda-forge
+- pyviz/label/dev
+- nodefaults
 
 packages: &pkgs
-- python =3.9
-- notebook >=6.5.2
-- colorcet >=3.0.1
-- dask >=2022.7.0
-- datashader >=0.14.3
-- holoviews >=1.15.3
-- bokeh >=2.4.3
-- spatialpandas >=0.4.6
-- pyarrow =8
-- pandas >=1.4.2
-- panel <1
-- bokeh <3
+- python=3.11
+- notebook<7
+- colorcet>=3.1.0
+- dask>=2024.5
+- datashader>=0.16.3
+- hvplot>=0.10.0
+- holoviews>=1.19.1
+- pandas>=2.2.2
+- panel>=1.4.4
+- bokeh>=3.4.1
+- spatialpandas=0.5.0a1
```

Comment:
I plan to release spatialpandas when the work for `dask-expr` has been done. See https://github.com/holoviz/spatialpandas/issues/146.

If this example needs a non-dev release, I can release it before the `dask-expr` support.

Reply:
We've waited for so long now. I think we can wait some more :)

Reply:
Do we need to wait for `dask-expr` support or we could configure dask to use the older approach?

Reply:
I guess we could? But then, that means we still have to continue using the `pyviz/label/dev` channel.

Reply:
> Do we need to wait for dask-expr support

No. Currently, the example works without dask-expr support. I think the dev version of spatialpandas is to remove some Pandas warnings, but I cannot remember. Azaya can you test out with the latest spatialpandas release and report back?

Reply:
This is the warning from using spatialpandas 0.4.10:
```
examples/nyc_buildings/envs/default/lib/python3.11/site-packages/spatialpandas/io/parquet.py:137: FutureWarning: Passing 'use_legacy_dataset' is deprecated as of pyarrow 15.0.0 and will be removed in a future version.
  dataset = ParquetDataset(
 ```